finality-aleph/src/lib.rs

```diff
@@ -59,6 +59,9 @@ pub struct SessionPeriod(pub u32);
 #[derive(Copy, Clone, Debug, Default, Eq, PartialEq, Hash, Ord, PartialOrd, Encode, Decode)]
 pub struct MillisecsPerBlock(pub u64);
 
+#[derive(Copy, Clone, Debug, Default, Eq, PartialEq, Hash, Ord, PartialOrd, Encode, Decode)]
+pub struct BaseUnitCreationDelay(pub u64);
```

Comment:
Please add a comment documenting what units we are using (milliseconds)

Reply:
Comment moved